portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,9 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function createListElement(text) {
```

Comment:
What is this function for? It seems like it's not being used yet.

Reply:
I think the function is for taking data from the servlet and pushing it to the front end